lib/suma/api/commerce.rb

```diff
@@ -336,6 +336,7 @@ class SimpleOrderHistoryEntity < BaseEntity
     expose :image, with: ImageEntity do |inst|
       inst.checkout.items.sample&.offering_product&.product&.images&.first
     end
+    expose :begin_fulfillment_at, &self.delegate_to(:checkout, :cart, :offering, :begin_fulfillment_at)
```

Comment:
Avoid exposing fields like this to the UI. The UI shouldn't have to map the concept of 'begin fulfillment at' to 'available for pickup at'. Do `expose :available_for_pickup_at, &self.delegate_to(:checkout, :cart, :offering, :begin_fulfillment_at)` instead. That way, if the meaning of anything involved changes (for example, let's say we start offering delivery), the API does not need to change.